running-code/execute-script.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
getting here after reading the `execute-package` lesson and omg i need to get better at not reading PRs in alphanumeric order and read them in semantic order lmao